content-security-policy/embedded-enforcement/allow_csp_from-header.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Embedded Enforcement: Allow-CSP-From header.</title>
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <script src="../support/testharness-helper.sub.js"></script>
+</head>
+<body>
+  <script>
+    var tests = [
+      { "name": "Same origin iframes are always allowed.", 
+        "origin": Host.SAME_ORIGIN,
+        "csp": "style-src 'unsafe-inline'; script-src 'unsafe-inline';", 
```

Comment:
Nit: Do you need the trailing `;` at the end of all of these? It's not necessary from a grammar perspective, so... :)